app/assets/stylesheets/components/_form.scss

```diff
@@ -53,3 +52,7 @@
 #chomp_session_invitees {
   display: none;
 }
+
+.text-danger {
```

Comment:
This will affect other texts with bootstrap class text-danger so best not to use it in real projects I think. I checked master and we have no other texts using it so it's ok!